packages/editor/src/components/editor-help/index.native.js

```diff
@@ -57,6 +57,18 @@ const HELP_TOPICS = [
 	},
 ];
 
+const kebabCaseSettings = {
+	splitRegexp: [
+		/([\p{Ll}\p{Lo}\p{N}])([\p{Lu}\p{Lt}])/gu, // One lowercase or digit, followed by one uppercase.
+		/([\p{Lu}\p{Lt}])([\p{Lu}\p{Lt}][\p{Ll}\p{Lo}])/gu, // One uppercase followed by one uppercase and one lowercase.
+	],
+	stripRegexp: /(\p{C}|\p{P}|\p{S})+/giu, // Anything that's not a punctuation, symbol or control/format character.
+};
+
+function kebabCase( string ) {
+	return paramCase( string, kebabCaseSettings );
+}
+
```

Comment:
We use the `kebabCase` to generate the slug of each `HELP_TOPICS` item. Since this content is static, defined here, only used within this file, and it's relatively small, I'm wondering if we could simply define the `slug` for each one, instead of dynamically calculating it 🤔.

Reply:
This is a good idea, thanks @fluiddot 👍 I'm happy to follow-up in another PR.

Reply:
Followed up in #48802.

Reply:
Thank you @tyxla for creating the PR 🙇 , I'll take a look at it soon.